tox.ini

```diff
@@ -8,32 +8,31 @@
 
 [tox]
 # PyPy configuration (on Linux/OSX):
-# - /usr/local/bin/pypy -> /opt/pypy2.7-v7.3.0-osx64/bin/pypy
 # - /usr/local/bin/pypy3 -> /opt/pypy3.6-v7.3.0-osx64/bin/pypy3
 envlist =
-    py{27,34,35,36,37,38,39,310}-wrapt{1.10,1.11,1.12,1.13}
-    pypy, pypy3
+    py{37,38,39,310}-wrapt{1.10,1.11,1.12,1.13,1.14}
+    py{311}-wrapt{1.14}
```

Comment:
1.14 is needed to get Python 3.11 working https://github.com/GrahamDumpleton/wrapt/issues/196